Composer/packages/lib/code-editor/src/components/toolbar/__tests__/ToolbarButtonMenu.test.tsx

```diff
@@ -35,7 +35,7 @@ const templatePayload: TemplateRefPayload = {
 const propertiesPayload: PropertyRefPayload = {
   kind: 'property',
   data: {
```

Comment:
Why are these tests being changed? They're not related to this work at all.

Reply:
They're being changed because they were failing mysteriously. I feel like the word "this" is too common to be used as a name and search target in tests like this without false positives happening.